crawl-ref/source/dat/des/variable/mini_monsters.des

```diff
@@ -2550,6 +2550,28 @@ MAP
  ...
 ENDMAP
 
+# A shopkeeper has learned why there are no NetHack-style shops in this dungeon.
+NAME:   nicolae_nh_ransacked_shop
+TAGS:   transparent patrolling no_item_gen
+DEPTH:  D:4-10
+KITEM:  1d = any armour no_uniq w:90 / any weapon no_uniq / nothing w:25
```

Comment:
20 items on average (even if mostly junk) seems a bit generous,  I'd switch the weight proportion to be 1/5 items and 4/5 floor. The band will have good items too (maybe they stole em??? could give the band gold to enhance the story).

Reply:
Noted! Giving the band members gold might be finicky, since if you specify gold explicitly you also have to make sure to explicitly specify their usual gear or else they just get the gold.